orttraining/orttraining/test/training_api/common/synthetic_data_loader.cc

```diff
@@ -38,32 +40,54 @@ void RandomInts(std::vector<IntType>& rets, IntType low, IntType high) {
 
 }  // namespace
 
+template <typename T>
+void SyntheticSampleBatch::AddIntInput(const std::vector<int64_t>& shape, T low, T high) {
+  data_vector_.emplace_back(SyntheticInput(shape));
+
+  std::vector<T> values(data_vector_.back().NumOfElements());
```

Comment:
>ata_vector_.back().NumOfElements() [](http://example.com/codeflow?start=25&length=34)

Do you really need to initialize with default values? Coding standards say reserve() + push_back()

Reply:
fixed.